build-tools/packages/build-cli/src/commands/modify/lockfile.ts

```diff
@@ -0,0 +1,96 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { updatePackageJsonFile } from "@fluidframework/build-tools";
+import { Flags } from "@oclif/core";
+import execa from "execa";
+
+import { BaseCommand } from "../../base";
+import { releaseGroupFlag } from "../../flags";
+
+/**
+ * Updates the version of a dependency in the lockfile.
+ *
+ * @remarks
+ * This command is primarily used manually to update transitive dependencies when we need to address CVEs.
+ *
+ * Note that this applies to all packages in the specified release group.
+ */
+export default class UpdateDependencyInLockfileCommand extends BaseCommand<
+	typeof UpdateDependencyInLockfileCommand
+> {
+	static readonly summary =
+		"Updates a dependency in the lockfile to the latest version of a specified semver range.";
+
+	static readonly description =
+		`Note that if the version passed in to the command is not within the range of versions ` +
+		`naturally accepted by the packages that depend on it, after this command runs the lockfile might not reflect the ` +
+		`version that was passed in, but the latest version that complies with the semver range declared by the dependent packages.`;
```

Comment:
Can't template strings have embedded newlines? Or does the formatter remove them?

Reply:
It doesn't, I just thought it would mess up the rendering of the description by oclif which breaks lines where it deems necessary. If I break them myself in the source inside a single template string we end up with this:

![image](https://github.com/microsoft/FluidFramework/assets/716334/dfb8a8ac-0746-4168-b0fb-aae94c9ac452)

Not the end of the world but I prefer the cleaner version on the left from letting oclif take care of that.